src/buildercore/bootstrap.py

```diff
@@ -457,7 +481,8 @@ def update_stack(stackname, service_list=None, **kwargs):
     service_update_fns = OrderedDict([
         ('ec2', update_ec2_stack),
         ('s3', update_s3_stack),
-        ('sqs', update_sqs_stack)
+        ('sqs', update_sqs_stack),
+        ('fastly', update_terraform_stack)
```

Comment:
This can't be `('fastly', update_fastly_stack)` because if we had Fastly and, say, Github managed by Terraform we would still have to `apply` them together. Or maybe there's a way to separate them. In any case, not an hard decision until we have some future additional service there.

Reply:
Also, right now all these are executed during deploy. We should move the default to only update EC2 during deploy as SQS/S3/Fastly/anything else should not be modified during deploy but with `update_template` (to be renamed, accepting suggestions?) by a human operator.